docs/kubefirst/local/install.md

```diff
@@ -19,36 +20,52 @@ To create a new Kubefirst cluster locally, run
+- [Certificate Issuers](../https://letsencrypt.org/certificates/) - Let's Encrypt browser-trusted certificates
+- [Cert Manager](../https://github.com/cert-manager/cert-manager) - Certificate Automation Utility
+- [Chart Museum](../https://github.com/helm/chartmuseum) - Helm Chart Registry
+- [External Secrets](../https://github.com/external-secrets/kubernetes-external-secrets) - Syncs Kubernetes secrets with Vault secrets
+- [GitHub Action Runner](../https://github.com/features/actions) - Self Hosted GitHub Action Runner
+- [Nginx Ingress Controller](../https://docs.nginx.com/nginx-ingress-controller/intro/overview/) - Ingress Controller
+- [Metaphor](../https://github.com/kubefirst/metaphor-frontend-template) - A sample app to demonstrate CI/CD in on Kubernetes. Contains Devlopment, Staging, and Production environments.
+- [Vault](../https://github.com/hashicorp/vault) - Secrets Management
 
-### Super powers user needs and Certificates to deal with HTTPS locally.
+### How to resolve HTTPS Certificate Warnings
 
-To increase the experience of Cloud running Kubefirst locally, we use [Mkcert](https://github.com/FiloSottile/mkcert) to generate local certificates and serve `https` with Linkerd Ingress Controller.
-During installation, we generate these certificates and push them to Kubernetes as secrets to attach to Ingress resources.
-Since the certificates are auto-assigned, the browser doesn't recognize them as trusted certificates. 
-To resolve the warning that the browser shows when you access one of your applications,  we recommend to run the command:
+To resolve the warning that the browser shows when you access one of your applications, run the command:
 ```shel
 mkcert -install
 ```
-We recommend that you run this command above to avoid asking for your password and that you can understand with transparency why a superuser password is requested.
+ We use [Mkcert](https://github.com/FiloSottile/mkcert) to generate local certificates and serve `https` with Linkerd Ingress Controller.
 
-This step will install the CA (Certificate Authority) of MkCert in your trusted store and will make the browser trust in certificates generated by your Kubefirst local install.
+During installation, Kubefirst generates these certificates and pushes them to Kubernetes as secrets to attach to Ingress resources. The browser does not recognize auto-assigned certificates as trusted certificates and will generate security errors. 
 
-### Atlantis and ngrok integration
+This step will install the CA (Certificate Authority) of MkCert in your trusted store and will allow the browser to trust in certificates generated by your Kubefirst local install.
 
-During cluster provisioning, Terraform communicates with the host machine to create the desired resources. When Atlantis is installed via Kubefirst, it will use ngrok to expose the Atlantis server to the internet via webhook.
+### Atlantis and ngrok integration
```

Comment:
one detail worth mentioning in this section is that the ngrok tunnel used for kubefirst local has a 2-hour expiration unless you create an account with ngrok. this expiration will prevent you from using automated infrastructure as code through atlantis, but the rest of the platform will continue to function beyond that ngrok evaluation period.